app/src/main/java/camp/nextstep/edu/tictactoe/TicTacToeViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package camp.nextstep.edu.tictactoe
+
+import android.view.View
+import androidx.databinding.BindingAdapter
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.domain.*
+
+class TicTacToeViewModel(game: Game = Game()) : ViewModel() {
+
+    private var game: Game
```

Comment:
```suggestion
    private val game: Game
```
별도로 game 객체를 새로 선언하는 부분이 없는 경우 val로 만들어주는 것이 좋습니다!
init scope에서 초기화를 진행해준다면 val로 하더라도 문제가 되지 않습니다 :)